test/JDBC/input/BABEL-SP_COLUMNS_MANAGED-dep-vu-prepare.sql

```diff
@@ -0,0 +1,6 @@
+CREATE PROC babel_sp_columns_managed_dep_vu_prepare_p1 @Catalog AS sys.SYSNAME = NULL, @Owner AS sys.SYSNAME = NULL, @Table AS sys.SYSNAME = NULL, @Column AS sys.SYSNAME = NULL, @SchemaType AS sys.SYSNAME = 0
```

Comment:
how did we get the list of tests to be added? All the functions/procedures modified with missing tests?

Reply:
We used upgrade_validation and sql_validation framework to list down all the objects with missing test cases. We then went through whole list manually to understand whether test cases are required. More details can be found at https://quip-amazon.com/PwbHAEdd9CYJ/Additional-Test-cases-required-for-BABEL-3315